MAUI/ImageEditor/shape.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: Shape Annotation in .NET MAUI ImageEditor control | Syncfusion
+description: Learn about the shape annotation feature of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
```

Comment:
Learn about the shape annotations feature of Syncfusion .NET MAUI Image Editor(SfImageEditor) control.

Reply:
Changed a description